build/AzurePipelineTemplates/CsWinRT-BuildOnly-Stage.yml

```diff
@@ -0,0 +1,380 @@
+stages:
+- stage: BuildOnly
+  displayName: Build for Release|x64
+  jobs: 
+  - job: Build
+    pool:
+      vmImage: windows-2019
+    timeoutInMinutes: 90
+    # https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#multi-job-configuration
+    strategy:
+      maxParallel: 10
+      matrix:
+       x64_Release:
+          BuildPlatform: 'x64'
+          BuildConfiguration: 'release'
+    steps:
+    - checkout: self
+      clean: true
+      persistCredentials: true
+    
+# Clone TestWinRT
+    - task: CmdLine@2
+      displayName: Clone TestWinRT
+      enabled: false
+      inputs: 
+        script: get_testwinrt.cmd
+        workingDirectory: $(Build.SourcesDirectory)
+
+# Download procdump
```

Comment:
Do you think we should reuse the common stuff by refactoring it to a separate file?

Reply:
like CsWinRT-Build-Steps or so?